packages/documentation/.storybook/preview.js

```diff
@@ -1,12 +1,20 @@
+import { extractArgTypes, extractComponentDescription, setStencilDocJson } from '@pxtrn/storybook-addon-docs-stencil';
+import docJson from '@swisspost/design-system-components/dist/docs.json';
+import * as Components from '@swisspost/design-system-components-react';
+import beautify from 'js-beautify';
+import React from 'react';
+import { renderToString } from 'react-dom/server';
```

Comment:
```suggestion
import { renderToStaticMarkup } from 'react-dom/server';
```

This function is similar to the `renderToString` function but does not create some react specific attributes.
See: https://reactjs.org/docs/react-dom-server.html#rendertostaticmarkup